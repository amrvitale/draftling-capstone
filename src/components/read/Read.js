import React from 'react';
import { Link } from 'react-router-dom'

class Read extends React.Component {
    render() {
        return (
            <div className="readOnlyDraftlings">
                <section class="placeholderForSelectedDraftling">
                  <p>Placeholder for read only version of selected Draftling...</p>
                  <Link to='/postfreeformcritique'>
                        <button type="submit">Write a freeform critique</button>
                  </Link> 

                  <Link to='/posttemplatecritique'>
                        <button type="submit">Write a templated critique</button>
                  </Link> 
                </section>

            </div>
        );
    }
}
export default Read;

//make it toggle one or the other and pop open the critique option