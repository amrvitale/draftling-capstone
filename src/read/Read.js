import React from 'react';
import { Link } from 'react-router-dom'

class Read extends React.Component {
    render() {
        return (
            <div className="readOnlyDraftlings">
                <section class="placeholderForSelectedDraftling">
                  <p>Placeholder for read only version of selected Draftling...</p>
                  <Link to='/choosecritique'>
                        <button type="submit">Write a critique</button>
                  </Link> 
                </section>

            </div>
        );
    }
}
export default Read;

//make it toggle one or the other and pop open the critique option