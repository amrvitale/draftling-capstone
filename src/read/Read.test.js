import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Read from './Read'

describe (`Read component`, () => {
    it('renders a .Read by default', () => {
        const wrapper = shallow(<Read />)
        expect (toJson(wrapper)).toMatchSnapshot()
    })

    
})