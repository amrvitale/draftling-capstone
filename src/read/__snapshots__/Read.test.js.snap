// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read component renders a .Read by default 1`] = `
<div
  className="readOnlyDraftlings"
>
  <section
    className="placeholderForSelectedDraftling"
  >
    <p>
      Placeholder for read only version of selected Draftling...
    </p>
    <Link
      to="/choosecritique"
    >
      <button
        type="submit"
      >
        Write a critique
      </button>
    </Link>
  </section>
</div>
`;
